src/main/java/gregtech/api/capability/impl/MultiblockRecipeLogic.java

```diff
@@ -88,7 +88,7 @@ protected IItemHandlerModifiable getInputInventory() {
     // Used for distinct bus recipe checking
     protected List<IItemHandlerModifiable> getInputBuses() {
         RecipeMapMultiblockController controller = (RecipeMapMultiblockController) metaTileEntity;
-        return controller.getAbilities(MultiblockAbility.IMPORT_ITEMS);
+        return new ArrayList<>(controller.getAbilities(MultiblockAbility.IMPORT_ITEMS, MultiblockAbility.IMPORT_DUAL));
```

Comment:
Imo, this shouldn't need a separate ability of its own but should just be both IMPORT_ITEMS and IMPORT_FLUIDS

Reply:
I can't use IMPORT_FLUIDS (whose type is IFluidTank) since a list of `IItemHandlerModifiable`s are expected, unless i changed the returned list type

Reply:
I more meant, I don't think `IMPORT_DUAL` exists since `getAbilities(IMPORT_ITEMS)` should automatically grab any dual handlers since they should have the `IMPORT_ITEMS` ability as well